gen/java/ai/visma/asgt/dataservice/v1/AppendDataRequest.java

```diff
@@ -0,0 +1,1034 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
all old java files are deleted and new files are created because directory structure changed to build package easily